experimental/css-has-pseudo/test/_browser.html

```diff
@@ -10,7 +10,7 @@
 	<!-- This is the real test stylesheet and has correct CORS attributes and http headers -->
 	<link rel="stylesheet" href="http://localhost:8081/test/browser.expect.css" crossorigin="anonymous">
 	<script src="http://localhost:8082/dist/browser-global.js"></script>
-	<script>cssHasPseudo(document, { observedAttributes: ['attrname'], debug: true, hover: true, forcePolyfill: true });</script>
+	<script>cssHasPseudo(document, { observedAttributes: ['attrname'], debug: false, hover: true, forcePolyfill: true });</script>
```

Comment:
`true` is very noisy because of how style invalidations work.

Best to set this to false for our own tests unless we are actually debugging the tests.